cleanlab_studio/studio/enrichment.py

```diff
@@ -19,7 +19,7 @@
 Replacement = Tuple[str, str]
 ROW_ID_COLUMN_NAME = "row_id"
 FINAL_RESULT_COLUMN_NAME = "final_result"
-TRUSTWORTHY_SCORE_COLUMN_NAME = "trustworthy_score"
+TRUSTWORTHY_SCORE_COLUMN_NAME = "trustworthiness_score"
```

Comment:
Should we update the variable to also be similar here? Might cause confusion if its referenced as both the trustworthy score and trustworthiness score

```suggestion
TRUSTWORTHINESS_SCORE_COLUMN_NAME = "trustworthiness_score"
```

Reply:
Good call. Change made.